views/layout.ejs

```diff
@@ -43,7 +43,7 @@ See the LICENSE file for copyright details
       <a href="/"><span class="mdi mdi-home"> Home</span></a>
       <a href="/docs/about.md"><span class="mdi mdi-information"> About</span></a>
       <a href="/docs"><span class="mdi mdi-file-document-multiple"> Docs</span></a>
-      <a href="https://opencollective.com/open-web-docs"><span class="mdi mdi-hand-coin"></span> Donate</title>
+      <a href="https://opencollective.com/open-web-docs"><span class="mdi mdi-hand-coin"> Donate</span></a>
```

Comment:
```suggestion
      <a href="https://opencollective.com/open-web-docs"><span class="mdi mdi-hand-coin"></span> Donate</a>
```

Reply:
Your suggestion is not consistent with other items in the navbar. All words are in side `<span>`s.

Reply:
They shouldn't be...

Reply:
Both render the same though. Ideally if the spans are solely for icons then there shouldn't be any text inside them. Do you want a new PR for this? or piggyback this with one of the next PRs.

Reply:
Yeah, the spans are meant solely for the icons -- if you could submit a new PR to move the text out of the spans, I'll merge it right away!